src/sqlancer/postgres/PostgresProvider.java

```diff
@@ -349,4 +336,55 @@ public String getDBMSName() {
         return "postgres";
     }
 
+
+    @Override
+    public String getQueryPlan(String selectStr, PostgresGlobalState globalState) throws Exception {
```

Comment:
It would be great if we could add some unit and/or integration tests before merging this.

Reply:
Thanks, and sorry for following up late. Do you think we could also specify a query plan string and test whether this method returns the expected result for that static string? Having such a second test would be very helpful.